### PR TITLE
update the readme.md

### PR DESCRIPTION
**Pull request by huideaki**
Add steps to use Frabric instead of Firebase and some known issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Crashlytics.Droid
 
-Firebase Crashlytics binding project for Android
+This binding provide support for Fabric Crashlytics and Firebase Crashlytics (Firebase is the prefered and easiest way to make this work).
```
Comment (MatFillion) [inline README.md]: provides

Comment (MatFillion) [reply README.md]: preferred

File: README.md [hunk 2]
```diff
@@ -16,15 +16,61 @@ Simply download the 5 AAR from maven and extract the classes.jar from each and r
...
+This will create an app project on Fabric (The binding is not able to create this for you).
+
+Next step is to include the library to your Xamarin.Android project.
+
+Add a meta data 
+``` xml
+<meta-data android:name="io.fabric.ApiKey" android:value="[YOUR-API-KEY]" />
+```
+
+or the prefered way in the Assembly info for environment control
```
Comment (MatFillion) [inline README.md]: preferred

File: README.md [hunk 3]
```diff
@@ -52,3 +98,7 @@ It's pretty strait forward.
 
 If you want, you can add a pre-build script to your project to get the manifest's "android:versionName" and set it right into the String.xml file, but that's up to you.
 
+## Known Issues
+The targeted Xamarin.Android project platform needs to be of Android 8.0 otherwise the application is not able to initialize with firebase.
```
Comment (MatFillion) [inline README.md]: needs to target Android 8.0 (API Level 26). 

I think the word target makes it clearer that we're talking about compilation and not the device.

File: README.md (hunk 1 above)
Comment (huideaki) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (huideaki) [reply README.md]: same

File: README.md (hunk 3 above)
Comment (huideaki) [reply README.md]: I agree, i added a better comment

**Pull request merged**